src/LanguageManager/LanguageEditView.php

```diff
@@ -0,0 +1,229 @@
+	 *
+	 * @return void
+	 */
+	private function print_text_fields( Language $language )
+	{
+		$text_fields = [
+			LanguagesTable::COLUMN_NATIVE_NAME,
+			LanguagesTable::COLUMN_ENGLISH_NAME,
+			LanguagesTable::COLUMN_HTTP_CODE,
+			LanguagesTable::COLUMN_ISO_639_1_CODE,
+			LanguagesTable::COLUMN_LOCALE,
+		];
+
+		foreach ( $text_fields as $field ) {
+			$label = $this->labels->label( $field );
+			$value = $language->name( $field );
+			?>
+			<tr>
+				<td class="alignright">
+					<label for="<?= esc_attr( $field ) ?>_id">
+						<?= esc_html( $label ) ?>
+					</label>
+				</td>
+				<td>
+					<input
+						type="text"
+						name="<?= esc_attr( $field ) ?>"
+						id="<?= esc_attr( $field ) ?>_id"
+						value="<?= esc_attr( $value ) ?>"
+					/>
+				</td>
+			</tr>
+			<?php
+		}
+	}
+
+	/**
+	 * HTTP priority numeric input
+	 *
+	 * @param Language $language
+	 *
+	 * @return void
+	 */
+	private function print_priority( Language $language )
+	{
+		?>
+		<tr>
+			<td class="alignright">
+				<label for="<?=esc_attr( LanguagesTable::COLUMN_PRIORITY )?>_id">
+					<?=esc_attr( $this->labels->label( LanguagesTable::COLUMN_PRIORITY ) )?>
+				</label>
+			</td>
+			<td>
+				<input
+					type="number" min="1" max="10"
+					name="<?=esc_attr( LanguagesTable::COLUMN_PRIORITY )?>"
+					id="<?=esc_attr( LanguagesTable::COLUMN_PRIORITY )?>_id"
+					value="<?=esc_html( $language->name( LanguagesTable::COLUMN_PRIORITY ) )?>"
+				/>
+			</td>
+		</tr>
+		<?php
+	}
+
+	/**
+	 * Checkbox for RTL property of a language
+	 *
+	 * @param Language $language
+	 *
+	 * @return void
+	 */
+	private function print_rtl( Language $language )
+	{
+		?>
+		<tr>
+			<td></td>
+			<td>
+				<input
+					type="checkbox"
+					name="<?=esc_attr( LanguagesTable::COLUMN_RTL )?>"
+					id="<?=esc_attr( LanguagesTable::COLUMN_RTL )?>_id"
+					value="1"
+					<?=checked( true, $language->is_rtl(), false )?>
+				/>
+				<label for="<?=esc_attr( LanguagesTable::COLUMN_RTL )?>_id">
+					<?=esc_html( $this->labels->label( LanguagesTable::COLUMN_RTL ) )?>
+				</label>
+			</td>
+		</tr>
+		<?php
+	}
+
+	/**
+	 * List of sites to assign a language
+	 *
+	 * @param \Inpsyde\MultilingualPress\Common\Type\Language $language
+	 *
+	 * @return void
+	 */
+	private function print_site_selector( Language $language )
+	{
+		$sites = get_sites();
+		$active_languages = get_available_languages( false );
+		?>
+		<tr>
+			<td class="alignright">
+				<?=esc_html_e( 'Sites', 'multilingualpress' )?>
+			</td>
+			<td>
+				<em><?=esc_html_e( 'Assign one or more sites in order to activate the language', 'multilingualpress')?></em>
+				<br />
+				<?php
+				/** @var \WP_Site $site */
+				foreach ( $sites as $site ) {
+					$active = $this->is_active_language_for_site( $language, $site, $active_languages );
+					?>
+					<label>
+						<input
+							type="checkbox"
+							name="sites[<?=esc_attr( $site->blog_id )?>]"
+							value="1"
+							<?php checked( true, $active )?>
```

Comment:
`checked()` has `true` as second argument default, so always pass the current thing as first one, then you only have to specify the expected value if it is anything else than `true`.